sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -311,29 +303,29 @@
     </item>
     <item>
       <EFDB_Name>relativeHumidity</EFDB_Name>
-      <Description>Relative Humidity</Description>
+      <Description>Relative Humidity.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>%</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>airTemperature</EFDB_Name>
-      <Description>Air Temperature</Description>
+      <Description>Air Temperature.</Description>
```

Comment:
I thought you were going to rename this to temperature? That's what the ticket says.

Reply:
I intended to and forgot. Done.